### DISCUSSION TITLE
How does the PNG plug-in handle a CHRM chunk?

### DISCUSSION THREAD

**Discussion by MarkCallow** [OpenImageIO/oiio#3079] [Q&A]
PNG CHRM chunks provide color space information (primaries, white point, black point). The oiio:ColorSpace attribute does not provide such information so what does the PNG plug-in do with it? Drop it on the floor? Or did I miss something?

**Comment by lgritz** [comment]
It probably does drop it on the floor. It would be great to have a patch that correctly pulls this data out of the PNG file and adds it as metadata to the ImageSpec that is returned that describes the image.

"oiio:ColorSpace" should just be a color space name (generally, one recognized by OpenColorIO), so you shouldn't cram primaries, etc. into that. We should make separately named metadata for those things. We can pick names and standardize across all the formats that support such things.
